roles/confluent.kafka_connect/defaults/main.yml

```
@@ -23,6 +23,20 @@ kafka_connect_default_internal_replication_factor: "{{ [ groups['kafka_broker']
 
 kafka_connect_group_id: connect-cluster
 
+kafka_connect_plugins_path: 
+- /usr/share/java
+kafka_connect_confluent_hub_install: "{{true if kafka_connect_confluent_hub_plugins|length >0 else false}}" 
```

This should evaluate to `true/false` on its own

```suggestion
kafka_connect_confluent_hub_install: "{{kafka_connect_confluent_hub_plugins|length >0}}" 
```